features/step_definitions/check_basecap.rb

```diff
@@ -0,0 +1,18 @@
+require 'open3'
+require 'json'
+
+def is_capability_enabled?(capability)
```

Comment:
Seems `def` is mostly defined in lib/ instead of features/step_definitions. @liangxia is this right?

Reply:
`clusterversion` is one of the resources in OpenShift. We can use `lib/openshift/cluster_version.rb` for functions related to `clusterversion`.

Reply:
@dpuniaredhat you can remove this step_definiton once #3624 is merged.  So your PR will just need that one line addition to the `serviceaccount.feature` file